rwa5_4/config/rwa5_sensor_kootbally1.yaml

```diff
@@ -0,0 +1,224 @@
+sensors:
+
+  breakbeam_0a:
+    type: break_beam
+    pose:
```

Comment:
I guess you mentioned about changing [a,b,c,d,e] to numbers ?! Fine with this also. Just wanted to make sure.

Reply:
This was an old reference file, now deleted